client/my-sites/post-share/index.jsx

```diff
@@ -203,121 +195,75 @@ class PostShare extends Component {
-		return (
-			<CompactCard className="post-share__footer-items" key={ index }>
-				<div className="post-share__footer-item">
-					<div className="post-share__handle">
-						<SocialLogo icon={ service === 'google_plus' ? 'google-plus' : service } />
-						<span className="post-share__handle-value">
-							{ service === 'twitter' ? `@${ externalName }` : externalName }
-						</span>
-					</div>
-					<div className="post-share__timestamp">
-						<Gridicon icon="time" size={ 18 } />
-						<span className="post-share__timestamp-value">
-							{ shareDate }
-						</span>
-					</div>
-					<div className="post-share__message">
-						{ message }
-					</div>
-				</div>
-				<EllipsisMenu>
-					<PopoverMenuItem icon="visible">
-						Preview
-					</PopoverMenuItem>
-					<PopoverMenuItem icon="pencil">
-						Edit
-					</PopoverMenuItem>
-					<PopoverMenuItem icon="trash">
-						Trash
-					</PopoverMenuItem>
-				</EllipsisMenu>
-			</CompactCard>
-		);
-	}
+	renderUpgradeNudge() {
+		if (
+			this.props.planSlug === PLAN_BUSINESS ||
+			! isEnabled( 'publicize-scheduling' )
+		) {
+			return null;
+		}
 
-	renderScheduledList() {
 		const {
-			planSlug,
-			translate,
-			businessRawPrice,
 			businessDiscountedRawPrice,
+			businessRawPrice,
+			translate,
 			userCurrency,
-			scheduledSharingActions,
-			siteId,
-			postId,
 		} = this.props;
 
-		if ( planSlug !== PLAN_BUSINESS ) {
-			return (
-				<Banner
-					className="post-share__footer-banner"
-					callToAction={
-						translate( 'Upgrade for %s', {
-							args: formatCurrency( businessDiscountedRawPrice || businessRawPrice, userCurrency ),
-							comment: '%s will be replaced by a formatted price, i.e $9.99'
-						} )
-					}
-					description={ translate( 'Live chat support and no advertising.' ) }
-					list={ [
-						translate( 'Live chat support' ),
-						translate( 'No Advertising' )
-					] }
-					plan={ PLAN_BUSINESS }
-					title={ translate( 'Upgrade to a Business Plan!' ) }
-				/>
-			);
-		}
-		return ( <div>
-			<QuerySharePostActions siteId={ siteId } postId={ postId } status="scheduled" />
-			{ scheduledSharingActions.map( ( item, index ) => this.renderFooterSectionItem( item, index ) ) }
-		</div> );
-	}
-
-	renderPublishedList() {
-		return 'published';
+		return (
+			<Banner
+				className="post-share__footer-banner"
+				callToAction={
+					translate( 'Upgrade for %s', {
+						args: formatCurrency( businessDiscountedRawPrice || businessRawPrice, userCurrency ),
+						comment: '%s will be replaced by a formatted price, i.e $9.99'
+					} )
+				}
+				description={ translate( 'Live chat support and no advertising.' ) }
+				list={ [
+					translate( 'Live chat support' ),
+					translate( 'No Advertising' )
+				] }
+				plan={ PLAN_BUSINESS }
+				title={ translate( 'Upgrade to a Business Plan!' ) }
+			/>
+		);
 	}
 
 	renderFooter() {
-		const { footerSection } = this.state;
+		if ( ! isEnabled( 'publicize-scheduling' ) ) {
```

Comment:
2955df0496e500e7480cbfed8d339920b4427121 and bd284ccc38375f964239ff5d26ecc4f1fc7a9ba9